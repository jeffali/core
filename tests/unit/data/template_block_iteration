[%CFEngine BEGIN %]
This is a block of text
That contains list variables: $(foo.list)
With text before and after.
[%CFEngine END %]
