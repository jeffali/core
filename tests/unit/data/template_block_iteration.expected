This is a block of text
That contains list variables: 1
With text before and after.
This is a block of text
That contains list variables: 2
With text before and after.
